src/Model/Post.php

```diff
@@ -125,6 +132,7 @@ public function __construct( \WP_Post $post ) {
 
 		parent::__construct( 'PostObject', $post, $restricted_cap, $allowed_restricted_fields, $post->post_author );
 		$this->init();
+		parent::prepare_fields();
```

Comment:
This is so more fields can be added in extended class before they are processed

Reply:
I think you might want to just create a NEW model in that case. . .maybe? 🤔

Reply:
Maybe worth discussing with @CodeProKid a bit too

Reply:
You can use the `graphql_return_modeled_data` filter for this -> https://github.com/wp-graphql/wp-graphql/blob/feature/model-layer/src/Model/Model.php#L401

Reply:
^ what @CodeProKid said. 

That filter will allow third party plugins, etc to extend the models as well. So they can define fields in one place, define their access checks for those fields, etc. . .then keep that separate from the resolvers

Reply:
Oh, I completely missed that filter. @CodeProKid @jasonbahl Thanks . I changed it back.